spec/formatting.md

```diff
@@ -173,13 +174,19 @@ and different implementations MAY choose to perform different levels of resoluti
 >   did not emit an error.
 > - Using a _variable_, the _resolved value_ of an _expression_
 >   could be used as an _operand_ or _option value_ if
->   calling the `getValue()` method of its _resolved value_ did not emit an error.
+>   calling the `getResult()` method of its _resolved value_ did not emit an error.
 >   In this use case, the `resolvedOptions()` method could also
 >   provide a set of option values that could be taken into account by the called function.
+>   - In some cases (such as in a number formatting function), `getResult()` would
+>     return the _operand_ of the function that returned the `MessageValue`. In other
```

Comment:
This is incorrect; for an expression like `{42 :number}` the function is expected to return the numerical value 42 rather than the literal string `'42'`, which is the operand here.

Also, please do use [semantic line breaks](https://sembr.org/) in spec text.

Reply:
I think the intention of this added paragraph is correct, but I think it could be a lot clearer.

Something like:

> - The value returned by `getResult()` might simply be the _operand_
>   of the function that returned the `MessageValue`,
>   or it might be some other value.
>   For example, for a function that extracts field from a data structure,
>   the result would be the extracted value,
>   rather than the _operand_ it was extracted from.

Reply:
I've edited this part; please take another look.